ecosystem.config.js

```diff
@@ -20,6 +20,12 @@ module.exports = {
       args: "start",
       cwd: path.resolve(__dirname, "shop/"),
     },
+    {
+      name: "dashboard",
+      script: "npm",
+      args: "start",
+      cwd: path.resolve(__dirname, "dashboard/"),
+    },
```

Comment:
**Issue**: The new 'dashboard' script configuration in PM2 lacks a restart policy which could lead to service downtime if the process exits unexpectedly. <br> **Fix**: Add a restart policy to the 'dashboard' configuration to improve the resilience of the service. <br> **Code Suggestion**: 
 ```
 +    {
 +      name: "dashboard",
 +      script: "npm",
 +      args: "start",
 +      cwd: path.resolve(__dirname, "dashboard/"),
 +      restart: 'on-failure'
 +    },
 ```